src/autostartup/modules/autostartup.lua

```diff
@@ -0,0 +1,3 @@
+package.path = package.path .. ";/modules/?;/modules/?.lua;/modules/?/init.lua"
+local dns = require("dns")
+
```

Comment:
It seems like the rest of this file's contents are missing? Ditto for every other package converted to a module - it seems like only the new modules have any actual code.

Reply:
-.- Not sure what happened here. I swear I had all the code added.

Reply:
I'll work on fix later today.